march_hardware_interface/config/march3/controllers.yaml

```diff
@@ -10,11 +10,18 @@ march:
       type: march_pdb_state_controller/MarchPdbStateController
       publish_rate: 50
     trajectory:
-      type: position_controllers/JointTrajectoryController
+      type: effort_controllers/JointTrajectoryController
       joints:
         - right_hip
         - left_hip
         - right_knee
         - left_knee
         - right_ankle
         - left_ankle
+      gains: # Required because we're controlling an effort interface
+        right_hip: {p: 50000, i: 5000, d: 5000, i_clamp: 10, publish_state: true, antiwindup: true}
+        left_hip: {p: 50000, i: 5000, d: 5000, i_clamp: 10, publish_state: true, antiwindup: true}
+        right_knee: {p: 50000, i: 5000, d: 5000, i_clamp: 10, publish_state: true, antiwindup: true}
+        left_knee: {p: 50000, i: 5000, d: 5000, i_clamp: 10, publish_state: true, antiwindup: true}
+        right_ankle: {p: 5000, i: 500, d: 5000, i_clamp: 10, publish_state: true, antiwindup: true}
+        left_ankle: {p: 5000, i: 500, d: 5000, i_clamp: 10, publish_state: true, antiwindup: true}
```

Comment:
?

Reply:
Still working on improving the PID gains, they will be tested when we do our final pullrequest. Or wasn't that the question?